build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '0.1.0'
+version '0.2.0'
```

Comment:
I would love to bump it to 1.0.0, anything below 1.0.0 seems a bit less serious to me.

Reply:
very early days for this lib, not need to rush imho.
pdf-client is already 4.0.0 :)